build-scripts/postinstall.js

```diff
@@ -44,11 +45,12 @@ if ( !isDependency ) {
 
 // Purge intermediate build files but leave the addon where it is
 shelljs.mv( addonAbsoluteName, repoPaths.root );
-shelljs.mv( path.join( addonAbsolutePath, "octbstack.dll" ), repoPaths.root );
+var dynlib = os.platform() === "win32" ? "octbstack.dll" : "liboctbstack.so";
+shelljs.mv( path.join( addonAbsolutePath, dynlib ), repoPaths.root );
```

Comment:
`liboctbstack.so` is not located in `addonAbsolutePath` on Linux. Only Windows works such that if a DLL file is in the same directory as the module that needs it then the module will find it. Linux uses `-Wl,-rpath`. So, at best, you could
```JS
if (os.platform() === "win32") {
  shelljs.mv( path.join( addonAbsolutePath, "octbstack.dll" ), repPaths.root );
}
```
and
```JS
if (os.platform() === "win32") {
  shelljs.mv( path.join( repoPaths.root, "octbstack.dll" ), addonAbsolutePath );
}
```
later on.